tests/integration/test_interaction.py

```diff
@@ -406,27 +438,33 @@ class AssessmentInteractionTest(DefaultAssessmentDataTestMixin, InteractionTestB
     All interactions are tested using mouse (action_key=None) and four different keyboard action keys.
     If default data changes this will break.
     """
-    @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
-    def test_item_no_feedback_on_good_move(self, action_key):
-        self.parameterized_item_positive_feedback_on_good_move(
-            self.items_map, action_key=action_key, assessment_mode=True
-        )
+    # @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
+    # def test_item_no_feedback_on_good_move(self, action_key):
+    #     self.parameterized_item_positive_feedback_on_good_move(
+    #         self.items_map, action_key=action_key, assessment_mode=True
+    #     )
+
+    # @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
+    # def test_item_no_feedback_on_bad_move(self, action_key):
+    #     self.parameterized_item_negative_feedback_on_bad_move(
+    #         self.items_map, self.all_zones, action_key=action_key, assessment_mode=True
+    #     )
 
     @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
-    def test_item_no_feedback_on_bad_move(self, action_key):
-        self.parameterized_item_negative_feedback_on_bad_move(
-            self.items_map, self.all_zones, action_key=action_key, assessment_mode=True
+    def test_move_items_between_zones(self, action_key):
+        self.parameterized_move_items_between_zones(
+            self.items_map, self.all_zones, action_key=action_key
         )
 
-    @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
-    def test_final_feedback_and_reset(self, action_key):
-        self.parameterized_final_feedback_and_reset(
-            self.items_map, self.feedback, action_key=action_key, assessment_mode=True
-        )
+    # @data(None, Keys.RETURN, Keys.SPACE, Keys.CONTROL+'m', Keys.COMMAND+'m')
+    # def test_final_feedback_and_reset(self, action_key):
+    #     self.parameterized_final_feedback_and_reset(
+    #         self.items_map, self.feedback, action_key=action_key, assessment_mode=True
+    #     )
 
-    @data(False, True)
-    def test_keyboard_help(self, use_keyboard):
-        self.interact_with_keyboard_help(use_keyboard=use_keyboard)
+    # @data(False, True)
+    # def test_keyboard_help(self, use_keyboard):
+    #     self.interact_with_keyboard_help(use_keyboard=use_keyboard)
```

Comment:
@haikuginger There isn't, thanks for catching this. I commented them out while debugging a specific test and then forgot to uncomment them :/.